packages/arb-token-bridge-ui/package.json

```diff
@@ -25,7 +25,7 @@
     "dayjs": "^1.10.6",
     "ethers": "^5.6.0",
     "fetch-retry": "^5.0.3",
-    "framer-motion": "^4.1.17",
+    "framer-motion": "^10.12.7",
```

Comment:
`useViewportScroll` being used from framer-motion@v4 has been deprecated. We should use `useScroll` and test the code.

<img width="696" alt="image" src="https://user-images.githubusercontent.com/7558499/236838998-e3df28e3-0776-4e95-a94d-56a3cfe56508.png">

Reply:
Sure , working on it